parachain/pallets/system/src/lib.rs

```diff
@@ -594,6 +568,29 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Force initializes the pallet if not already initialized.
+		///
+		/// Creates 2 agents (BridgeHub and AssetHub).
+		/// Creates 3 channels (Primary and secondary governance channels, BridgeHub channel).
+		///
+		/// Privileged. Can only be called by root.
+		///
+		/// - `origin`: Must be root
+		/// - `own_para_id`: The parachain id of the parachain housing this pallet. Most likely
+		///   BridgeHub.
+		/// - `asset_hub_para_id`: The parachain id of AssetHub.
+		#[pallet::call_index(10)]
+		#[pallet::weight((T::WeightInfo::force_initialize(), DispatchClass::Operational))]
+		pub fn force_initialize(
+			origin: OriginFor<T>,
+			own_para_id: ParaId,
```

Comment:
`own_para_id` seems unnecessary which can be retrieved from [runtime](https://github.com/Snowfork/polkadot-sdk/blob/00e8eb2e04375260d728f143352e21cbf7a4436c/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L337).

Reply:
Disagree, actually, the runtime is configured with the para_id using the genesis config for the ParachainInfo pallet.